### PR TITLE
Make source parameter not required

### LINKED ISSUE #28: `source` parameter for payments does not fall back to `base_seed` from configuration
The docs suggest that the `base_seed` configuration parameter will be used to determine the transaction source for payments, but the `source` param is still required anyway (https://github.com/stellar/bridge-server/blob/0.0.12/src/github.com/stellar/gateway/protocols/bridge/payment.go#L63) and there certainly doesn't appear to be any code for the fallback anyway (https://github.com/stellar/bridge-server/blob/0.0.12/src/github.com/stellar/gateway/protocols/bridge/payment.go#L128)

As a side note: what is the point of that config param if this doesn't work?

### PR DESCRIPTION
**Pull request by bartekn**
Close #28

### CODE REVIEW

File: src/github.com/stellar/gateway/bridge/handlers/request_handler_payment_test.go [hunk 1]
```diff
@@ -402,6 +406,61 @@ func TestRequestHandlerPayment(t *testing.T) {
...
+			Convey("it should return success", func() {
+				statusCode, response := net.GetResponse(testServer, validParams)
+				responseString := strings.TrimSpace(string(response))
+
+				assert.Equal(t, 200, statusCode)
+				expected := test.StringToJSONMap(`{
+				  "hash": "ad71fc31bfae25b0bd14add4cc5306661edf84cdd73f1353d2906363899167e1",
+				  "ledger": 1988728
+				}`)
+				assert.Equal(t, expected, test.StringToJSONMap(responseString))
```
Comment (nullstyle) [inline src/github.com/stellar/gateway/bridge/handlers/request_handler_payment_test.go]: Does this even work (i.e does it give good test output when it fails)?  I thought goconvey requires that you use its own assertions.

Comment (bartekn) [inline src/github.com/stellar/gateway/bridge/handlers/request_handler_payment_test.go]: Yes, it does. The first argument passed: `t` variable is `testing.T`.

**Comment by nullstyle** [comment]
+1.  But if you could, please hold off on further changes.  I'm currently in the middle of a big refactor of the bridge server to fix some issues but more importantly make testing a lot easier, and any changes outside of this refactor is going to cause a bunch of conflicts.

**Comment by bartekn** [comment]
Sure!

**Pull request merged**